content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Cut Service Mesh Overhead with Istio Ambient Mesh"
```

Comment:
Two notes:

* this is written from the perspective of "use it today", which I believe is Solo's recommendation, but should not be the Istio project recommendation.  It should be toned down in the edit.
* you may get SEO problems or challenges if you have a blog that is basically just a reposted version of https://www.solo.io/blog/reduce-cloud-cost-istio-ambient-mesh/.  

In general,  I'd encourage you to substantially differ from the Solo blog.  We never adopted them, but if we had, the [Istio blog guidelines](https://github.com/macruzbar/istio.io/blob/dfe2fc0b33a830155199622633f5bc2c57f2ce4c/content/en/blog/README.md) state "The Istio blog only publishes original content; we do not re-post content that has been posted elsewhere".

That said, I find this content compelling and my opinion is that we should be posting something very much like this whenever the time is right.  I am not appropriately in the loop to comment on if that time is now.

Reply:
+1 to what craig box said. Istio project recommendation is to NOT jump and use this today. Solo, as vendor, can say whatever they see fit.  Same thing for any home page promo around ambient right now when its not prime time ready. @linsun needs to understand that project website and vendor website are very different and treating them same is very dangerous to the community, users and project overall.

Reply:
@craigbox can you help point out where the blog is written as "use it today"? It is mentioned multiple times (beginning and conclusion) that ambient is alpha in the blog.  Happy to correct or clarify as needed to make this clear.